### PR TITLE
feat: add discussion template

### PR DESCRIPTION
**Pull request by sgoudham**
Making use of [discussion templates](https://docs.github.com/en/discussions/managing-discussions-for-your-community/creating-discussion-category-forms) which is now in limited public beta

### CODE REVIEW

File: .github/DISCUSSION_TEMPLATE/port-requests.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+title: "Name of the application/tool/website in lower-kebab-case (E.g. NeoVim -> neovim)"
```
Comment (backwardspy) [inline .github/DISCUSSION_TEMPLATE/port-requests.yml]: this rule makes sense on repo names, but perhaps not for discussion titles? i'd suggest using the brand's capitalisation (where applicable) which seems to be the natural inclination on our current discussions.

File: .github/DISCUSSION_TEMPLATE/port-requests.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before you submit your port request, **please read our [submission](https://github.com/catppuccin/catppuccin/blob/main/docs/contributing.md#submission) guidelines!**
```
Comment (backwardspy) [inline .github/DISCUSSION_TEMPLATE/port-requests.yml:6]: consider including "guidelines" in the link text too

File: .github/DISCUSSION_TEMPLATE/port-requests.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  - type: checkboxes
+    attributes:
+      label: Will you make the port?
+      description: The port has a much higher chance of completion if you are willing to get involved!
+      options:
+        - label: Yes, I'm willing to make the port
```
Comment (backwardspy) [inline .github/DISCUSSION_TEMPLATE/port-requests.yml:12]: do we need to provide a negative option too, or do we get one by default?

ᵒʳ ᵃʳᵉ ʷᵉ ᶠᵒʳᶜᶦⁿᵍ ᵉᵛᵉʳʸᵒⁿᵉ ᵗᵒ ᵐᵃᵏᵉ ᵗʰᵉᶦʳ ᵒʷⁿ ᵖᵒʳᵗ

Comment (sgoudham) [reply .github/DISCUSSION_TEMPLATE/port-requests.yml:12]: The checkbox isn't required so they can leave it unchecked or check it!

**Review by backwardspy — APPROVED** [review]
lgtm

**Pull request merged**